### ISSUE TITLE
Switch .net 472 version of SDK to use HttpClient instead of HttpWebRequest

### ISSUE THREAD

**Issue by MichaelSabin-Citrix** [aws/aws-sdk-net#3449]
### Describe the feature

With the upgrade to 4.0.0, I would like to recommend that the SDK use a consistent HTTP Pipeline for .Net Framework and Standard.  This will ease compatibility testing for customers who want to upgrade their existing app to .net standard

### Use Case

Compatibility between .Net Framework / Standard, also less testing since consistent SDK is used

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### AWS .NET SDK and/or Package version used

AWS SDK 4.0.0-preview

### Targeted .NET Platform

.Net Framework 4.7.2

### Operating System and version

Windows 11, Windows Server 2022

**Comment by normj** [comment]
@MichaelSabin-Citrix I would love to use `HttpClient` for both .NET Framework and .NET Core instead of using `HttpWebRequest` for .NET Framework. It would make our lives easier maintaining the SDK. We tried that in the past but since `HttpClient` is really a wrapper around `HttpWebRequest` in .NET Framework with limitations the performance was terrible when used at scale. We got a lot of customer issues and with their application performance degrading so we had to revert the change and is why today we use `HttpWebRequest` for .NET Framework and `HttpClient` for .NET Core.
